spec/plans/provision.fmf

```diff
@@ -203,3 +204,22 @@ example: |
                     model: 67
               - cpu:
                     model: 69
+
+/multihost:
+    summary: Multi-host testing specification
+
+    description: |
+        As a part of the provision step it is possible to assign
+        a role using ``role`` parameter for multi-host testing.
+        A role can be either client or server. Other multi-host
+        than server-client are not supported. Either both roles
```

Comment:
So for multi-host there can be only 1 client and 1  server?

Reply:
Correct, we discussed this and identified this a the most important scenario (even though beaker supports multiple servers and clients scenarios). Later we plan to actually extend multi-host testing to support also so-called orchestrator role to allow multi-host testing without mutual synchronization .

Reply:
The only reason at the time being is that we want to support current multihost tests using sync library [*] and to be able to do that we have to set SERVERS and CLIENTS variables. Hence we need to know exactly who is the server and who is the client. 

[*] with that it would be also possible to support all server-client test cases using beaker rhts-sync-* tools.

Reply:
But I suppose we can support more generic approach not limiting ``role`` just to "client" or "server" and we can allow any IDs (and hence not only 1-1 client-server scenarios but n-m scenarios). However, at the moment we don't have any tools to actually use other than 1-1 client-server scenarios). I do not have any objections to go this more generic direction.

Reply:
Updated to support arbitrary roles (not just server and client).